src/test/java/controllers/DefectController.java

```diff
@@ -24,19 +24,18 @@ public Response addDefect(String projectCode, Defect defect) {
                 .extract().response();
     }
 
-    public Response getDefects(String projectCode) {
+    public Response getDefect(String projectCode, int defectId) {
         return given()
                 .when()
                 .log().all()
-                .get("/v1/defect/" + projectCode)
+                .get("/v1/defect/" + projectCode + "/" + defectId)
                 .then()
                 .log().all()
-                .statusCode(SC_OK)
                 .extract().response();
     }
 
-    public Response updateDefect(String projectCode, int defectId, String newTitle) {
-        return given()
+    public void updateDefect(String projectCode, int defectId, String newTitle) {
```

Comment:
Why the return type is void here? It should return Response object, shouldn't it?